docs-conceptual/azps-11.6.0/azureps-support-lifecycle-lts.md

```diff
@@ -0,0 +1,161 @@
+
+## Versioning
+
+Azure PowerShell follows [Semantic Versioning][semantic-versioning] for version numbering. Semantic
+versioning is a system that uses a three-part number format: `major.minor.patch`.
+
+- **Major versions:** Breaking changes, including architectural and fundamental design changes.
+- **Minor versions:** Backwards compatible feature updates and improved Azure services coverage.
+- **Patch versions:** Backwards compatible security fixes without new features or functionality
+  changes.
+
+### Module version types
+
+Releases of Azure PowerShell modules can include the following types of versions:
+
+- **Generally available:** Modules version 1.0.0 and higher without _preview_ in the version.
+  Adheres to breaking change policy.
+- **Preview:** Modules less than version 1.0.0. Don't adhere to breaking change policy.
+- **Feature preview:** Modules version 1.0.0 and higher with _preview_ in the version. Don't adhere
+  to breaking change policy.
+
+Preview and Feature preview versions aren't recommended for production environments.
+
+## Release cadence
+
+Azure PowerShell is updated monthly, with two annual releases introducing breaking changes. This
+release cadence is designed to provide customers with predictable release dates, empowering
+businesses, developers, and the community to make strategic decisions when planning their roadmaps.
+
+## Support of releases
+
+Azure PowerShell ships two types of generally available releases. Customers can choose Long Term
+Support (LTS) or Standard Term Support (STS) releases. The quality of all releases is the same. The
+only difference is the length of support.
+
+- **Long Term Support (LTS) releases**
+
+  - Beginning with Az 12.0.0, even-numbered major versions are LTS releases.
+  - The final version of each LTS release is supported until the second subsequent STS version is
+    released.
+  - Supported for 18 months from the original major version release date.
+
+- **Standard Term Support (STS) releases**
+
+  - Odd-numbered major versions are STS releases.
+  - STS versions are supported until the next major release of Azure PowerShell.
+  - Supported for six months from the original major version release date.
+
+> [!IMPORTANT]
+> You must install the latest minor and patch version to qualify for support.
+
+The following table provides examples of how the release schedule and support work together:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
+| -------------- | ---------------- | ---------------- | ------------ | ------------ | ----------------------- |
+| 12.0.0         | May 2024         | ![Yes][1]        | Major        | LTS          | 12.1.0                  |
+| 12.1.0         | June 2024        |                  | Minor        |              | 12.2.0                  |
```

Comment:
In here, should we add all "LTS" , because 12.1.0,12.2.0 ... is also LTS version, 
For STS , 13.1.0 , 13.2.0 .. all STS. 


![image](https://github.com/MicrosoftDocs/azure-docs-powershell/assets/108926104/087b290e-85e8-4eef-b141-35869d57448d)

Reply:
Good point. I've made the suggested changes.